File: programmers/etc/level2/H-index.py
def solution(citations):
    answer = 0
    citations.sort()
    for i in range(len(citations)):
        if citations[i] >= len(citations)-i:
            answer += 1
    return answer


citations = [3, 0, 6, 1, 5]
print(solution(citations)) # 3

File: Java/programmers/etc/level2/H_Index.java
package CodeTest.Java.programmers.etc.level2;

import java.util.*;

public class H_Index {
    public int solution(int[] citations) {
        int answer = 0;
        Arrays.sort(citations);
        
        for (int i=0; i<citations.length; i++) {
            if (citations[i] >= citations.length-i) {
                answer++;
            }
        }
        
        return answer;
    }
}
